### PR TITLE
Making hearder

### PR DESCRIPTION
**Pull request by PriscilliaAmmeux**

### CODE REVIEW

**Review by Azahires — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+  "keywords": [],
+  "author": "Wild Code School",
+  "license": "ISC",
+  "devDependencies": {
+    "concurrently": "^7.0.0",
+    "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-icons": "^4.6.0"
+  }
```
Comment (LoicBrassart) [inline package.json]: Remove this, it has nothing to do in this package.json (global)

File: package.json [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-  },
-  "keywords": [],
-  "author": "Wild Code School",
-  "license": "ISC",
-  "devDependencies": {
-    "concurrently": "^7.0.0",
-    "husky": "^7.0.0"
-  }
-}
+{
```
Comment (LoicBrassart) [inline package.json:1]: Note (nothing to do) : This file is 100% modified, that's quite unsettling... Encoding error, maybe ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please fix the global package file

**Review by Azahires — APPROVED** [review]

**Review by Azahires — DISMISSED** [review]

**Review by CharlyJuteau — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Better ! :ok_hand:

**Review by CharlyJuteau — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
Saw no Evil

**Review by CharlyJuteau — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**